CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.4.1 (2019-11-11)
```

Comment:
Should we update the date to 2019-11-12 since this didn't make it out on Monday?